projects/storefrontstyles/scss/cxbase/blocks/buttons.scss

```diff
@@ -45,7 +45,9 @@ button {
     max-height: 55px;
     min-width: 55px;
   }
-
+  &-to-configuration {
```

Comment:
`buttons.scss` is a file with our customizations of bootstrap. Please move our new component styles to a separtate file in the `components/` folder (it makes sense to nest it in a new `/configurator` folder)

Note: please remember to add the selector name in the list in the `_sparta-theme.scss` file then